spec/GameSpec.js

```diff
@@ -0,0 +1,111 @@
+describe('Game', function() {
+
+  var game;
+
+  beforeEach(function () {
+    game = new Game();
+  }); 
+
+  it('can roll a gutter game', function() {
+
+    rollMany(0,20);
+    expect(game.score()).toEqual(0);
+
+  });
+
+  it('can roll a game on ones', function() {
+
```

Comment:
Be consistent with line spacing - you don't have breaks below, why do you have them here? (or vice versa)